src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -375,13 +419,27 @@ def get_suggestion(self, random_search=False):
         total_params : dict
             parameter suggestion
         """
+        if self.parallel and len(self.total_data)>20 and len(self.running_data):
```

Comment:
Why hardcode >20 here?

Reply:
Because in the original TPE, they will randomly sample 20 config for exploring and initialize.
I want to maintain this logic.

Reply:
Got it~